client/network/src/service/out_events.rs

```diff
@@ -81,12 +81,9 @@ pub struct Sender {
 	/// Name to identify the channel (e.g., in Prometheus and logs).
 	name: &'static str,
 	/// Number of events in the queue. Clone of [`Receiver::in_transit`].
```

Comment:
I don't understand the second half of this comment.

Reply:
Yeah, there is an error in the comment. It should be `Receiver::queue_size`.